bat.d/41-framework

```diff
@@ -0,0 +1,536 @@
+#!/bin/sh
+# Battery Plugin TEMPLATE for laptops providing standard sysfs nodes for
```

Comment:
change to: 
`# 65-framework - Battery Plugin for Framework laptops w/ framework_laptop driver in the out of tree version from https://github.com/DHowett/framework-laptop-kmod`